pros-sys/src/misc.rs

```diff
@@ -2,11 +2,14 @@ use core::ffi::*;
 
 pub const NUM_V5_PORTS: usize = 21;
 // v5 comp
-pub const COMPETITION_DISABLED: i32 = 0b001;
```

Comment:
are you sure that these are supposed to be u8?

Reply:
`competition_get_state` returns a u8 bitfield so presumably its members are u8's

Reply:
See: https://pros.cs.purdue.edu/v5/api/c/misc.html#competition-get-status

Reply:
ok, cool.